router/route.go

```diff
@@ -13,19 +11,16 @@ import (
 
 // Routes load middlewares
 func Routes(middleware ...gin.HandlerFunc) http.Handler {
-	var (
-		router    = gin.Default()
-		jwtSecret = os.Getenv("JWT_SECRET")
-	)
+	var router = gin.Default()
```

Comment:
I also prefer negroni for middleware handling 😐 .  Sorry to be so specific.   If you feel strongly about sticking with gin, then I'll roll with it.